README.md

```diff
@@ -1,5 +1,7 @@
 # klaviyo-android-sdk
-
+![Latest](https://jitpack.io/v/klaviyo/klaviyo-android-sdk.svg)
+![Releases](https://github.com/klaviyo/klaviyo-android-sdk/releases)
```

Comment:
Looks like a broken image link in the preview? Or will that be populated later

Reply:
im hoping itll be populated later...

Reply:
ops youre right this is def broken